src/controllers/users.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const {
+  getAll,
+  addUser,
+  getUser,
+  deleteUser,
+  updateUser,
+} = require('../services/usersServer');
+
+function getAllAction(req, res) {
+  const users = getAll();
+
+  res.send(users);
+};
+
+function addUsersAction(req, res) {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(422);
+
+    return;
+  }
+
+  addUser(name);
+
+  res.statusCode = 201;
+  res.send('Added');
+}
+
+function getUserAction(req, res) {
+  const { userId } = req.params;
+
+  if (!userId) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  const user = getUser(userId);
+
+  if (!user.length) {
```

Comment:
Instead of checking for 'user.length', you should check if the 'user' is not null or undefined

Reply:
done